app.js

```diff
@@ -59,6 +59,7 @@ router.get('/people', function(req, res) {
         {name: 'swkim', url: 'assets/swkim.jpg'},
         {name: 'bwpark', url: 'assets/bwpark.jpg'},
         {name: 'jskim', url: 'assets/jskim.jpg'}
+        ,{name: 'Jongmin', url:'assets/jongmin.jpg'}
```

Comment:
I like your "comma first" style. Cheers!

Reply:
Can you also use the file name consistently (e.g., Jongmin.jpg vs jongmin.jpg), so we could avoid any potential confusion ?

Reply:
Thanks for your cheering and have in mind that consistent character to avoid potential.

Reply:
Can you update this PR with the change, so I can merge into master?